Milestone 1: Project complete

peterrobert opened the pull request:
project ready for review

peterrobert committed:
added main background image

peterrobert committed:
navigation complete

peterrobert committed:
added footer images

peterrobert committed:
project finished

peterrobert committed:
html and css validation complete

peterrobert committed:
installed stickler

peterrobert committed:
fixed stickler issue

peterrobert committed:
updated the read me file

peterrobert committed:
updated the stickler

peterrobert committed:
ignoting font-awesome

peterrobert committed:
removed font-awesome since not used

peterrobert committed:
error Unexpected whitespace before ":"  fixed

peterrobert committed:
fixing indentation error

rOluochKe reviewed (changes requested):
@peterrobert 
### This is impressive, good job so far, however, there are few changes you need to make for the project to be approved
- Write a more descriptive pull request showing what you have done on the project [check this guide for more info](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- write a more detailed readme file explaining what the project is all about, technologies, license, contributors, who I can install the project [check this guide](https://github.com/microverseinc/readme-template)
- Make sure your project passes linting both HTML and CSS [check this guide](https://github.com/microverseinc/linters-config/tree/master/css)
- Your navigation is still off from the original design, no logo, the gradient is still off and the search 
![navigation](https://user-images.githubusercontent.com/8114528/73034684-a7b52e80-3e56-11ea-9f4d-153b8629a54e.PNG)

- Always remember you can ask for help from slack, your team, mentor and or any TSE
### Once you make all these changes, resubmit your project for a review. I wish you all the best

peterrobert commented:
**Building-with-Backgrounds-and-Gradients Milestone 1: Complete Project**

In this milestone, I have completed the following:

- Created and validated the HTML mark up.

- Created and validated the CSS mark up.

- Added font-awesome and used some of its elements on my mark up

- Added stickler in order to fix common CSS issues

peterrobert committed:
improving readme file by addung a screenshot

peterrobert committed:
personal information added to readme file

peterrobert committed:
improved the usage section on the readme file

peterrobert committed:
improved the authors section on the read me file

peterrobert committed:
added font-awesome back

peterrobert committed:
ignoring font-awesome in stickler

peterrobert committed:
ignoring fonts folder

peterrobert committed:
fixed linter style indentation issue

peterrobert committed:
Fixed linters style issues

peterrobert committed:
fixed background gradient error

peterrobert committed:
updated the readme file to correct site link

euqueme commented on index.html:
```diff
@@ -6,16 +6,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <!-- ====fonts ==== -->
```
Delete commented code

euqueme commented on index.html:
```diff
@@ -6,16 +6,85 @@
+    <!-- ====fonts ==== -->
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu&display=swap" rel="stylesheet">
     <!-- === links === -->
```
**Delete commented code**

euqueme commented on index.html:
```diff
@@ -6,16 +6,85 @@
+        </section> <!-- .main-body -->
+    </main>
 
-    <!-- ==== font awesome cdn ===== -->
```
Delete commented code

euqueme commented on index.html:
```diff
@@ -6,16 +6,85 @@
 
-    <!-- ==== font awesome cdn ===== -->
-    <script src="https://use.fontawesome.com/da489ccde9.js"></script>
+    <!-- ===== footer section === -->
```
Delete commented code

euqueme commented on style.css:
```diff
@@ -0,0 +1,179 @@
+  padding-top: 10px;
+  padding-bottom: 10px;
+  text-transform: capitalize;
+  background-image: linear-gradient(#333, #5c5c5c);
```
[Optional] try this gradient instead
background-image: linear-gradient(#9c9c9c 20%,#5c5c5c, #4c4c4c);

euqueme commented on index.html:
```diff
@@ -6,16 +6,85 @@
+        <div class="footer-details">
+            <div class="shop-online">
+                <h5>Shop the Apple Online Store (1-800-MY-APPLE), visit an Apple Retail Store, or find a reseller.</h5>
+                <h5>Copyright © 2014 Apple Inc. All rights reserved. Terms of UsePrivacy Policy</h5>
```
The font-weight of these lines should be less than 500

euqueme reviewed (changes requested):
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) 

### Hello @peterrobert  :bulb: 
### I'm really impressed by the good work you've done so far :clap: thank you for making the changes the previous TSE requested, there are still a couple of mandatory requirements left before we can give you the approval:

- [ ]  The text in your footer shouldn't be bold

 **General**

- [ ] Comments: It’s better to delete commented code

[OPTIONAL] your navbar grid is darker than the original maybe you could play with some different combinations to achieve a more similar look (I have left an example in the commented code for you to try.)

#### I have commented on the lines of code that need to be fixed but its been a blast to review this project. :clinking_glasses:

### Please :pray: correct these changes and submit back for a code review when you are done :rocket:

#### Happy Coding! :woman_technologist:

STATUS OF THE PROJECT: REQUESTED CHANGES :traffic_light:

peterrobert committed:
changes made according to review

peterrobert commented:
All changes have been done as asked.

euqueme reviewed (approved):
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) :woman_technologist:

### Hii @peterrobert :handshake: 

I downloaded your project and I notice you made all the changes :raised_hands:   but the Live Demo doesn't reflect them :thinking: 
![image](https://user-images.githubusercontent.com/52209067/73124218-70ca4000-3f5e-11ea-9fe7-0987a9ec7a27.png)

### Please correct this :memo:  and submit the project right away no need to ask for a code review anymore

STATUS OF THE PROJECT: APPROVED :green_apple:
